packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/utils/menuItemIsAllowed.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { menuItemIsAllowed } from './menuItemIsAllowed';
+
+const isAllowedHandlerMock = jest.fn();
+const testUrlMap = new Map<string, string>([
+    ['/testUrl1', '/testPolicyResource1'],
+    ['/testUrl2', '/testPolicyResource2'],
+]);
+
+describe('menuItemIsAllowed', () => {
+    beforeEach(() => {
+        isAllowedHandlerMock.mockClear();
+    });
+
+    it('returns true when url is undefined', () => {
```

Comment:
Maybe it.each to have parametrized tests and reduce the amount of test code?

Reply:
Doesn't make it more readable I think.